daemon/bitcoin/helpers_test.go

```diff
@@ -0,0 +1,392 @@
+package bitcoin
+
+import (
+	"bytes"
+	"encoding/hex"
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/btcsuite/btcd/btcutil/psbt"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightningnetwork/lnd/lntypes"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+var psbtExample = "cHNidP8BAH0CAAAAAbxLLf9+AYfqfF69QAQuETnL6cas7GDiWBZF+3xxc/Y/AAAAAAD+////AofWEgAAAAAAIgAgvGKLsRKUcp0qk/lkYWpzGJQi51RkG5J51NwHb6B6Hh+1If0jAQAAABYAFL+6THEGhybJnOkFGSRFbtCcPOG8AAAAAAABAR8wBBAkAQAAABYAFHemJ11XF7CU7WXBIJLD/qZF+6jrAAAA"
```

Comment:
do you mind breaking this super big lines? for example:
```go
	testPsbt   = "cHNidP8BAF4CAAAAAebcw2XfHoopKx18ULWF0I7AnoofStu5fYK6Gw/h3N5KAQAA" +
		"AAD9////Adp25zsAAAAAIgAg9q16iQ9ibL8Qh2iOsKXARie3hM32ui9Fp3bAj86aq9PiAAA" +
		"AAAEBK2Z35zsAAAAAIgAg3xr4GtUMUHo8/3EiFNMfpTb/WlTc4YeVqtm78aDSWtsBBWSpFI" +
		"Kk4PsTu5sqL5J8ju/Onlbd4kF5h2MhAqnfNT0ODvVOmREaPFRsvVAzcNw/ILgASvJMDvL9F" +
		"qN/ZwLiALF1IQMi5J5CowMrFZslrnQxnk8UJZRTdEHxgASs69YHnhzcHmisAAA="
```

Reply:
done 🏁